src/library_browser.js

```diff
@@ -1268,7 +1268,7 @@ mergeInto(LibraryManager.library, {
       }
       var data = msg.data['data'];
       if (data) {
-        if (!data.byteLength) data = new Uint8Array(data);
+        data = new Uint8Array(data);
```

Comment:
ok, now i see.

please add asserts on that `assert(data instanceof ArrayBuffer)`, good for for the check, and makes the code clearer